client-app/src/pages/account/router/index.ts

```diff
@@ -7,14 +7,16 @@ import AccountOrders from "@account/views/account-orders/index.vue";
 import AccountPayments from '@account/views/account-payments/index.vue';
 import AccountUsers from "@account/views/account-users/index.vue";
 import { accessDeniedUrl } from '@common/constants';
+import Features from "@common/features"
 import Permissions from "@common/permissions"
 
 Vue.use(VueRouter);
 
-
-const beforeEnterWithPermissions = (to: any, from: any, next: any, ...permissions: string[]) => {
-  const authResult = Vue.$can( ...permissions);
-  if(authResult){
+// eslint-disable-next-line
+const beforeEnterWithPermissions = (to: any, from: any, next: any, featureName: string, ...permissions: string[]) => {
```

Comment:
Better rename this function. Since this checks not only permissions but also the activity of the feature.

Reply:
Or extract to separate function checkFeatureActivity(to, from, featureName).